tests/deployment_test.go

```diff
@@ -1172,19 +1172,20 @@ func (s *DeploymentSuite) TestSetCurrent_BeforeAndAfterRegister() {
 
 	// list should say it's current (with some delay)
 	s.EventuallyWithT(func(t *assert.CollectT) {
+		a := assert.New(t)
 		list, err := s.FrontendClient().ListDeployments(ctx, &workflowservice.ListDeploymentsRequest{
 			Namespace: s.Namespace(),
 		})
-		require.NoError(t, err)
+		a.NoError(err)
 		found, isCurrent1 := 0, false
 		for _, d := range list.GetDeployments() {
 			if d.Deployment.BuildId == dep1.BuildId {
 				found++
 				isCurrent1 = d.IsCurrent
 			}
 		}
-		require.Equal(t, 1, found)
-		require.True(t, isCurrent1)
+		a.Equal(1, found)
```

Comment:
@dnr it seems require calls FailNow which panics. So it's not compatible with EventuallyWithT